AndroidWeek1/app/src/main/java/org/sopt/androidweek/sign/SignInViewModel.kt

```diff
@@ -0,0 +1,33 @@
+package org.sopt.androidweek.sign
+
+import android.util.Log
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import kotlinx.coroutines.launch
+import org.sopt.androidweek.repository.dto.request.SignInRequest
+import org.sopt.androidweek.repository.network.SampleCreator
+
+class SignInViewModel : ViewModel() {
+    val loginStatus = MutableLiveData(false)
```

Comment:
loginStatus가 Two-way data binding에 사용되는 것이 아니라면 외부에 공개해주는 변수/뷰모델 내부에서 값을 바꾸는 Mutable 객체로 구성하는 것이 더 안전합니다.

```suggestion
    private val _loginStatus = MutableLiveData(false)
    val loginStatus: LiveData<Boolean> = _loginStatus
```